xbmc/addons/binary/interfaces/api1/PVR/AddonCallbacksPVR.h

```diff
@@ -237,6 +240,7 @@ class CAddonCallbacksPVR : public ADDON::IAddonInterface
   static void UpdateEpgEvent(const EpgEventStateChange &ch, bool bQueued);
 
   CB_PVRLib    *m_callbacks; /*!< callback addresses */
+  static std::atomic_bool m_enabled;
```

Comment:
adding a specific variable like pvrEnabled to a generic base class violates the concept of hierarchy.